verifier.sh

```diff
@@ -222,24 +220,7 @@ sig_hand () {
 
         echo "Destroying [$lxc_name] lxc"
         if [ "$LXC_DESTROY" = "lxc-destroy" ]; then
```

Comment:
We should remove all this old logic an just use:

```sudo $LXC_KILL -n $lxc_name```

See also lines 223-225 and 637-639